src/reducers/articles.js

```diff
@@ -1,16 +1,36 @@
 import * as ActionType from '../actions/articles';
+import { camelizeKeys } from 'humps';
 
-function articlesReducer (state = [], action) {
+function taggedArticles(state = {}, action) {
+    switch (action.type) {
+        case ActionType.LOADED_TAGGED_ARTICLES_SUCCESS:
+            if (action.response && action.response.text) {
+                let response = JSON.parse(action.response.text);
```

Comment:
thanks for your code review :+1: 
will add ```try ... catch ``` and make it pass eslint 

really appreciated  :sob: